eclair-core/src/main/resources/reference.conf

```
@@ -191,63 +191,101 @@ eclair {
+        ratios {
+          base = 0.0
+          cltv = 0.05             // when computing the weight for a channel, consider its CLTV delta in this proportion
+          channel-age = 0.4       // when computing the weight for a channel, consider its AGE in this proportion
+          channel-capacity = 0.55 // when computing the weight for a channel, consider its CAPACITY in this proportion
+        }
+
+        // Virtual fee for additional hops
+        // Corresponds to how much you are willing to pay to get one less hop in the payment path
+        hop-cost {
+          fee-base-msat = 500
+          fee-proportional-millionths = 200
+        }
+
+        mpp {
+          min-amount-satoshis = 15000 // minimum amount sent via partial HTLCs
+          max-parts = 5 // maximum number of HTLCs sent per payment: increasing this value will impact performance
+        }
+
+        percentage = 0 // set all experiments to 0% of the traffic by default
+      }
+
+      // One section per experiment
+      // The percentages of the traffic for different experiments must sum to 100.
+      // After starting an experiment, if you want to modify it you should give it a new name. If you don't, then you'll
+      // end up with metrics for different sets of parameters under the same name.
+      experiments {
+        control = ${eclair.router.path-finding.default} {
+          percentage = 100 // 100% of the traffic use the default configuration
+        }
+
+        // Examples of other configs as 0% experiments:
+
+        // To optimize for fees only:
+        test-fees-only = ${eclair.router.path-finding.default} {
+          ratios {
+            base = 1
+            cltv = 0
+            channel-age = 0
+            channel-capacity = 0
+          }
+          hop-cost-base-msat = 0
+          hop-cost-millionths = 0
```

Don't you need to update the format of `hop-cost` to reflect what's in `default`?
Same for other experiments below.

```suggestion
          hop-cost {
            fee-base-msat = 0
            fee-proportional-millionths = 0
          }
```

---

You're right. That's a problem with overriding a default conf, if you make a typo it won't complain and will silently use the default.